### ISSUE TITLE
How to run locally from source?

### ISSUE THREAD

**Issue by axu2** [FooSoft/mangle#32]
How do I run this locally from source? Windows or Mac, either is fine. My background includes web development in Python3 Flask. venv, etc

**Comment by axu2** [comment]
Figured it out. 

I installed the 64 bit version from here:
https://www.lfd.uci.edu/~gohlke/pythonlibs/#pyqt4

```
>pip freeze
Pillow==6.2.2
PyQt4 @ file:///C:/Users/.../GitHub/mangle/PyQt4-4.11.4-cp27-cp27m-win_amd64.whl
reportlab==3.5.59
```

**Comment by axu2** [comment]
Solved by #34
